src/main.rs

```diff
@@ -44,8 +45,12 @@ async fn main() {
         panic!("Could not validate the supplied configurations: {e}")
     }
 
+    if let Some(true) = config.prometheus_metrics {
+        start_prometheus_server();
+    }
+
     let graph_node_endpoint = config.graph_node_endpoint.clone();
-    let private_key = config.private_key.clone();
+    let private_key = config.private_key.clone().unwrap().clone();
```

Comment:
can replace this with the helper like `&config.wallet_input().unwrap().to_string()`